app/controllers/application_controller.rb

```diff
@@ -1,7 +1,7 @@
 class ApplicationController < ActionController::Base
-  include Pundit
   protect_from_forgery with: :exception
 
+  include Pundit
```

Comment:
Small thing, but includes are usually at the top of the class.

Reply:
Alrighty thanks I'll be sure to remember that next time :) I've updated this and will merge with master. Thanks Ben!